.travis.yml

```diff
@@ -18,14 +18,14 @@ before_deploy:
 deploy:
 - provider: script
   skip_cleanup: true
-  script: "./gradlew publish closeAndReleaseRepository -s -Pversion=$TRAVIS_TAG -Psigning.keyId=$SIGNING_KEY_ID -Psigning.password=$SIGNING_PASSWORD -Psigning.secretKeyRingFile=$TRAVIS_BUILD_DIR/secring.gpg"
+  script: "./gradlew publishToNexus closeAndReleaseRepository -s -Pversion=$TRAVIS_TAG -Psigning.keyId=$SIGNING_KEY_ID -Psigning.password=$SIGNING_PASSWORD -Psigning.secretKeyRingFile=$TRAVIS_BUILD_DIR/secring.gpg"
```

Comment:
Should not `closeAndReleaseRepository` task be removed?

Reply:
No, we still want to close and release the staging repository.

Without the `nexus-publish-plugin`, oss.sonatype.org implicitly creates a staging repository for us which we then want to close and release using the `gradle-nexus-staging-plugin`.
On Travis, we had the problem that multiple staging repositories were created because the Travis server farm uses multiple public IP addresses.

With the `nexus-publish-plugin`, we explicitly create a staging repository and then deploy to this specific staging repository. After that, we still need to close and release this repository.